content/post/117-js-ipfs-0.53.md

```diff
@@ -65,19 +65,19 @@ A few caveats:
 2. It only works over the loopback address (not `localhost`) as loopback is considered a [secure context](https://w3c.github.io/webappsec-secure-contexts/), otherwise it would require [TLS][]
 3. Only the `ipfs.add` API has full-duplex streaming enabled for this release - expect a lot more in future!
 
-There's a new example called [ipfs-client-add-files](https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-client-add-files) that you can use as a sandbox to experiment with this new client.
+There’s a new example called [ipfs-client-add-files](https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-client-add-files) that you can use as a sandbox to experiment with this new client.
 
-If you fire it up, you'll see progress and file import events appearing before the upload request has completed, something that's not been possible with the `ipfs-http-client` until now.
+If you fire it up, you’ll see progress and file import events appearing before the upload request has completed, something that’s not been possible with the `ipfs-http-client` until now.
 
-What's next? We're going to extend the [gRPC][] implementation to all streaming methods and get a server into `go-IPFS` so you can use the same client with that implementation too.
+What’s next? We”re going to extend the [gRPC][] implementation to all streaming methods and get a server into `go-IPFS` so you can use the same client with that implementation too.
```

Comment:
@jennwrites did you mean to turn this into a double smart quote?

Reply:
Oops no!